cms/articles/models.py

```diff
@@ -358,8 +358,15 @@ def previous_version(self, request: "HttpRequest", version: int) -> "TemplateRes
         # NB: Little validation is done on previous_version, as it's assumed handled on save
         revision = get_object_or_404(self.revisions, pk=correction.value["previous_version"])
 
+        revision_object = revision.as_object()
```

Comment:
non-blocking nitpick: `revision_object` -> `page_from_revision` (or `revision_page`).

LGTM otherwise

Reply:
Great suggestion, changed in [f0942e8](https://github.com/ONSdigital/dis-wagtail/pull/203/commits/f0942e8ab66695b271ffb1f59992a611b0f8547f)